packages/core-connector-interface/package.json

```diff
@@ -7,10 +7,10 @@
   "license": "GPL-3.0-only",
   "files": [
     "src/",
-    "!*.spec.ts",
-    "!*.spec.d.ts",
-    "!*.spec.js",
-    "!*.spec.js.map"
+    "!**/*.spec.ts",
```

Comment:
Why is this in the logging PR?

Reply:
Went through all packages to fetch their latest package.json / tsconfig.json etc and unified them.

Reply:
Not logging related?

Reply:
Yes!

Reply:
Yes!